pkg/controller/nodeipam/ipam/cidrset/cidr_set_test.go

```diff
@@ -836,12 +836,24 @@ func TestCIDRSetv6(t *testing.T) {
 func TestCidrSetMetrics(t *testing.T) {
 	cidr := "10.0.0.0/16"
 	_, clusterCIDR, _ := netutils.ParseCIDRSloppy(cidr)
+	clearMetrics(map[string]string{"clusterCIDR": cidr}, true)
+
 	// We have 256 free cidrs
 	a, err := NewCIDRSet(clusterCIDR, 24)
 	if err != nil {
 		t.Fatalf("unexpected error creating CidrSet: %v", err)
 	}
-	clearMetrics(map[string]string{"clusterCIDR": cidr})
+
+	clusterMaskSize, _ := clusterCIDR.Mask.Size()
+	max := 1 << uint32(24-clusterMaskSize)
```

Comment:
if we are going to do this I think we should avoid we drift in a future, we can create

```
func maxCidrs(clusterMaskSize, clusterMaskSize) int {
}
```
add an unit test for that function and use it for the other unit tests

Reply:
Since we're going to need this function in `cidr_set.go` and `mult_cidr_set.go`, is it okay if a new file `pkg/controller/nodeipam/ipam/utils.go` is created and the function is placed in that file? similarly, i see a lot of functions like `getIndexForCIDR()`, `indexToCIDRBlock()`, `getBeginningAndEndIndices()` shared between these two files which can be moved to the new file. wdyt?

Reply:
I think that duplicating this small code at this time is ok